### ISSUE TITLE
"Plot Cross Section" Not Working

### ISSUE THREAD

**Issue by vondeckerstein** [NorthArrowResearch/xsviewer-issues#65]
## The Problem

XS will not plot cross sections, though the database seems to have loaded correctly.

## Reproduction steps
![XSViewer Bug Report](https://user-images.githubusercontent.com/9078110/108927871-cf049f00-7606-11eb-9f6b-8d558e1dbc78.PNG)

1. I followed steps here: https://xsviewer.northarrowresearch.com/Tutorials/1_viewing_exporting_xs.html
2. No Cross Sections Populate in the "Cross Section Selections" Dialog

## Exception message

None Given

## Anything else?

Screenshot attached. Running Windows 10, unsure if other software dependencies exist.

**Comment by philipbaileynar** [comment]
How big is your database? Can you zip it and attach it to this ticket or email it to info@northarrowresearch.com please.

**Comment by vondeckerstein** [comment]
Phillip,

The database is about 400 MB zipped and about 1 GB unzipped, and was provided by Dr. John Shelley. I can ask him if I can share his google drive with you. Playing with the program further, it seems like I came into the instructions out of order. I did not have any data "Selections" created, which seems to be a pre-requisite I did not know about. Having a few Selections created, it seems to be working fine now but throws the following error still:

I would consider it "closed" for my purposes, unless you would like any further information to fix that error popup.

Thank you!

![Cross Section Viewer Error 2](https://user-images.githubusercontent.com/9078110/109035595-6bbe4f80-768e-11eb-978a-f45126b03991.PNG)
